CMakeLists.txt

```diff
@@ -331,6 +331,9 @@ install(PROGRAMS $<TARGET_FILE:llvm-as>
 add_custom_target(clang_links DEPENDS clang)
 add_custom_command(TARGET clang_links POST_BUILD
   COMMAND ${CMAKE_COMMAND} -E create_symlink ${CLANG_BIN_DIR}/lib/clang/6.0.0/lib/linux/libclang_rt.builtins-x86_64.a ${PROJECT_BINARY_DIR}/lib/libclang_rt.builtins-x86_64.a
+  COMMAND ${CMAKE_COMMAND} -E create_symlink ${CLANG_BIN_DIR}/bin/clang-6.0 ${PROJECT_BINARY_DIR}/bin/clang
```

Comment:
should we create a link to ${CLANG_BIN_DIR}/bin/clang instead?  otherwise we'll have to update this whenever the version gets updated.

Reply:
We have ${CLANG_BIN_DIR}/lib/clang/6.0.0/lib/linux/libclang_rt.builtins-x86_64.a hardcoded, so we would have to update that manually too.

Reply:
yes, that would be nice